src/solc_0.7/common/BaseWithStorage/ERC721BaseToken.sol

```diff
@@ -18,6 +18,10 @@ contract ERC721BaseToken is IERC721, WithSuperOperators, WithMetaTransaction {
     bytes4 internal constant ERC165ID = 0x01ffc9a7;
     bytes4 internal constant ERC721_MANDATORY_RECEIVER = 0x5e8bf644;
 
+    uint256 internal constant NOT_ADDRESS = 0xFFFFFFFFFFFFFFFFFFFFFFFF0000000000000000000000000000000000000000;
+    uint256 internal constant OPERATOR_FLAG = (2**255);
```

Comment:
for consistency/completeness, maybe we should add here `BURNED_FLAG = (2**160)`

Reply:
yes, that would be better